db/migrate/20190212111202_create_policies.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePolicies < ActiveRecord::Migration
+  def change
+    create_table :policies do |t|
+      t.string :name
+      t.string :version
+      t.binary :schema, limit: 16.megabytes
+      t.references :account, index: true, foreign_key: { on_delete: :cascade }, limit: 8
```

Comment:
There should also be a uniqueness on `['name', 'version', 'tenant_id']`

Reply:
@hallelujah I am checking that on https://github.com/3scale/porta/pull/593/commits/65156364a402eace60e8aa415e919a685b3c3a7f
I was doubting if putting this in the DB too because in theory if the tenant goes very fast doing the same action twice, the DB would catch it but the code wouldn't 🤔 So I will add it 👍